deployments/with-creds/ci/values.yaml

```diff
@@ -32,6 +32,27 @@ concourse:
                 app: ci-web
                 release: ci
     sidecarContainers:
+      - name: telegraf
+        image: telegraf
+        volumeMounts:
+        - name: dsdsocket
+          mountPath: /var/run/datadog
+        command:
+        - /bin/bash
+        - -c
+        - |
+          echo '
+          [[inputs.prometheus]]
+            urls = ["http://127.0.0.1:9391"]
+            metric_version = 2
+            name_override = "concourse.ci"
+            [inputs.prometheus.tags]
+              environment = "ci"
+          [[outputs.datadog]]
+            url = unix:///var/run/datadog/dsd.socket
```

Comment:
This needs to be quoted, but also won't work for a couple reasons @jomsie and I discovered while YOLO deploying to prod today:
* the datadog output plugin for telegraf _requires_ the `apikey` parameter
* the datadog output plugin for telegraf fails if `url` has the `unix://` scheme

So the only way to use telegraf for this purpose is if we directly submit our metrics to the datadog API. Based on

> If you’re submitting metrics directly to the Datadog API without using DogStatsD, expect
> ...
> * The full payload is approximately ~ 100 bytes. However, with the DogStatsD API, compression is applied, which reduces the payload size.

https://docs.datadoghq.com/api/v1/metrics/#submit-metrics

it sounds like we might see increased network i/o on the web nodes if we continue with this strategy.

Reply:
hi @jamieklassen , I haven't seen any change after the comment above, are you going with the path of `directly submit our metrics to the datadog API`?

Reply:
yeah that part actually got implemented in greenpeace. This PR can probably just be closed assuming y'all are comfortable waiting to cut over the whole deployment process to greenpeace.

Reply:
so if I understand correctly, this PR is for updating hush-house metrics for wavefront, which is already done in greenpeace. And it still requires some work in this PR. 

Is there a timeline of migrating hush-house metrics to wavefront? @scottietremendous